src/Console/ConsoleOptionParser.php

```diff
@@ -576,6 +576,7 @@ public function addSubcommand($name, array $options = [])
             $command = $name;
             $name = $command->name();
         } else {
+            $name = Inflector::underscore($name);
```

Comment:
Looking into the code a bit more it looks like it is not possible to have multiple methods set up, the method itself always had to be camelCase:

    $method = Inflector::camelize($command);

So this is mainly for making the calling and setup part more consistent.